resources/webfonts/readme.md

```diff
@@ -0,0 +1,8 @@
+# Webfonts
+
+This is a script which analyzes fonts used in the application and prepares metrics data for script-based language fonts to be used within.
+
+## Usage
+
+When you added script-based font — they are usually for left-to-right languages, such as Arabic, Hebrew etc. — ensure you called the script to enable your application handle metrics properly.
+Pass the root directory of application as the only parameter or ensure it is the current dir and call with no params. 
```

Comment:
It would be helpful to understand where the difference is visible, so you can check whether things were run correctly. Should we add a testing program or something?

Reply:
I.e. describe how to compile and check that things work.

Reply:
Ok, done.